updates/35/updates.sql

```diff
@@ -639,4 +639,370 @@ INSERT INTO assets(identifier, name, type) VALUES('SBTC', 'Super Bitcoin', 'W');
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0x9EaF8C1E34F05a589EDa6BAfdF391Cf6Ad3CB239', 'yearn.finance', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:8453/erc20:0x9EaF8C1E34F05a589EDa6BAfdF391Cf6Ad3CB239', 'A', 8453, '0x9EaF8C1E34F05a589EDa6BAfdF391Cf6Ad3CB239', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:8453/erc20:0x9EaF8C1E34F05a589EDa6BAfdF391Cf6Ad3CB239', 'YFI', 'yearn-finance', 'YFI', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:250/erc20:0x29b0Da86e484E1C0029B56e817912d778aC0EC69', 'yearn.finance', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:250/erc20:0x29b0Da86e484E1C0029B56e817912d778aC0EC69', 'A', 250, '0x29b0Da86e484E1C0029B56e817912d778aC0EC69', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:250/erc20:0x29b0Da86e484E1C0029B56e817912d778aC0EC69', 'YFI', 'yearn-finance', 'YFI', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:10/erc20:0x9046D36440290FfDE54FE0DD84Db8b1CfEE9107B', 'yearn.finance', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:10/erc20:0x9046D36440290FfDE54FE0DD84Db8b1CfEE9107B', 'A', 10, '0x9046D36440290FfDE54FE0DD84Db8b1CfEE9107B', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:10/erc20:0x9046D36440290FfDE54FE0DD84Db8b1CfEE9107B', 'YFI', 'yearn-finance', 'YFI', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:324/erc20:0x27d0A2b5316b98088294378692F4EAbfB3222e36', 'ZeroLend', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:324/erc20:0x27d0A2b5316b98088294378692F4EAbfB3222e36', 'A', 324, '0x27d0A2b5316b98088294378692F4EAbfB3222e36', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:324/erc20:0x27d0A2b5316b98088294378692F4EAbfB3222e36', 'ZERO', 'zerolend', 'ZERO', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x6985884C4392D348587B19cb9eAAf157F13271cd', 'LayerZero', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x6985884C4392D348587B19cb9eAAf157F13271cd', 'A', 56, '0x6985884C4392D348587B19cb9eAAf157F13271cd', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x6985884C4392D348587B19cb9eAAf157F13271cd', 'ZRO', 'layerzero', 'ZRO', NULL, NULL, NULL);
```

Comment:
the started timestamp seem to be missing in all the assets you added :( was it due to rate limit or something?